android/keyple-plugin/android-cone2/src/main/java/org/eclipse/keyple/plugin/android/cone2/Cone2PluginImpl.java

```diff
@@ -109,7 +120,9 @@ public void setParameter(String key, String value) {
     @Override
     protected SeReader fetchNativeReader(String name) throws KeypleReaderException {
         // Returns the current reader if it is already listed
-        for (SeReader reader : readers) {
+        //OD : use iterator instead of for loop
+        for (Iterator<SeReader> it = readers.iterator(); it.hasNext();){
```

Comment:
Quel est l’intérêt d'utiliser des itérateurs ici ?

Reply:
C'est a priori plus "thread safe" d'après Olivier.